features/verblijfstitel.feature

```diff
@@ -1,80 +1,78 @@
     - een verblijfstitel wordt NIET geleverd wanneer Datum einde verblijfstitel (39.20) vandaag is of een datum in het verleden heeft
 
     @gba
     Scenario: verblijfstitel heeft geen datum einde
-      Gegeven de persoon met burgerservicenummer '555550001' heeft de volgende 'verblijfstitel' gegevens
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende 'verblijfstitel' gegevens
       | aanduiding verblijfstitel (39.10) | datum ingang verblijfstitel (39.30) |
       | 37                                | 20210315                            |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
-      | fields              | verblijfstitel.aanduiding.code  |
+      | burgerservicenummer | 000000012                       |
+      | fields              | verblijfstitel.aanduiding       |
       Dan heeft de response een persoon met alleen de volgende 'verblijfstitel' gegevens
-      | naam            | waarde |
-      | aanduiding.code | 37     |
+      | naam                    | waarde                                                                              |
+      | aanduiding.code         | 37                                                                                  |
+      | aanduiding.omschrijving | code Vw 2000 art. 8, onder e, gemeenschapsonderdaan econ. niet-actief, arbeid spec. |
 
     @gba
     Abstract Scenario: verblijfstitel heeft <titel>
-      Gegeven de persoon met burgerservicenummer '555550001' heeft de volgende 'verblijfstitel' gegevens
+      Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende 'verblijfstitel' gegevens
       | aanduiding verblijfstitel (39.10) | datum einde verblijfstitel (39.20) | datum ingang verblijfstitel (39.30) |
       | 37                                | <datumEinde>                       | 20210315                            |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
+      | burgerservicenummer | 000000024                       |
       | fields              | verblijfstitel.aanduiding.code  |
       Dan heeft de response een persoon met alleen de volgende 'verblijfstitel' gegevens
-      | naam            | waarde |
-      | aanduiding.code | 37     |
+      | naam                    | waarde                                                                              |
+      | aanduiding.code         | 37                                                                                  |
+      | aanduiding.omschrijving | code Vw 2000 art. 8, onder e, gemeenschapsonderdaan econ. niet-actief, arbeid spec. |
```

Comment:
In de Achtergrond gegeven staat 'Vw 2000 art. 8,...' Ik denk dat het woord 'code' hier teveel is